include/z64.h

```diff
@@ -1689,6 +1689,13 @@ typedef struct DebugDispObject {
     /* 0x2C */ s32 pad; //Padding not in the OOT version
 } DebugDispObject; // size = 0x30
 
+typedef struct {
+    Actor* unk_00;
+    f32 unk_04;
+} z_sub_unkStruct_8013E640;
```

Comment:
I think this should just be an EnTk struct

Reply:
So you want z_sub_s to include the EnTk header? Why?